index.html

```diff
@@ -16,15 +18,25 @@ <h1 class="nav-brand">My Logo</h1>
         </a>
         <nav class="header-nav">
             <ul class="nav-list">
-                <li class="nav-item"><a href="#" >Portoflio</a></li>
-                <li class="nav-item"><a href="#" >About</a></li>
-                <li class="nav-item"><a href="#" >Contact</a></li>
+                <li class="nav-item"><a href="#">Portoflio</a></li>
+                <li class="nav-item"><a href="#">About</a></li>
+                <li class="nav-item"><a href="#">Contact</a></li>
             </ul>
         </nav>
-        <img src="images/icons/Union.svg" alt="Menu" />
+        <div id="toggle-menu" class="hidden">
+            <img src="images/icons/close.svg" alt="Close"  onclick="toggleMenu()"/>
+            <nav class="toggle-nav">
+                <ul class="nav-list">
+                    <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'Portoflio')">Portoflio</a></li>
+                    <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'About')">About</a></li>
+                    <li class="nav-item"><a href="#" onclick="mobileMenuOption(event,'Contact')">Contact</a></li>
+                </ul>
+            </nav>
```

Comment:
- [ ] Hi @isaacpitwa, nice job :+1:  for fixing the previous issue highlighted by the previous code reviewer. Once more, your close button is not visible. See the screenshots below:

Your Implementation | Figma Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/156243932-cf3be258-fd4c-4b09-8af3-07a5f0ccfede.png) | ![](https://user-images.githubusercontent.com/63087491/156243937-662fb4e6-3e95-496d-9106-d92253b6861b.png)

Please also see into the spacings of the menu options.